### PR TITLE
Extract changes from repositories on bump

### PR DESCRIPTION
**Pull request by Itxaka**
**What this PR does / why we need it**:
To know what version bumps are associated with a repo bump as its not clear on a simple view.

This should at least open the PR with the follwoing boyd:

```
Changes in quay.io/kairos/packages-arm64:
Updated versions:
[New] kcrypt/fips-static 0 -> 0.7.0
[New] kcrypt-challenger/fips-static 0 -> 0.5.0
[New] kairos-agent/fips-static 0 -> 2.1.8
[New] immucore/fips-static 0 -> 0.1.2
[Update] kairos-agent/fips 2.1.8 -> 2.1.9
[Update] kairos-agent/system 2.1.8 -> 2.1.9
[Update] kubeadm/k8s 1.21.12 -> 1.22.11
[Update] kubeadm/k8s 1.21.12 -> 1.22.9
[Update] kubeadm/k8s 1.21.12 -> 1.23.8
```

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: .github/repochanges/main.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+							Category: p.Category,
+							From:     oldpkg.Version,
+							To:       p.Version,
+							New:      false,
+						})
+					}
+				}
+			}
+
+			// Sort by newer package first
```
Comment (Itxaka) [inline .github/repochanges/main.go:71]: Im sure we could do this on pure bash, but the pure bash + earthly on top was killing me

File: .github/repochanges/main.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	"text/template"
+)
+
+func main() {
+
+	app := &cli.App{
+		Name:      "repochanges",
+		Version:   "0.0.1",
+		Authors:   []cli.Author{{Name: "Kairos authors"}},
+		Usage:     "Extract changes betweeen 2 luet repository version list files",
```
Comment (mauromorales) [inline .github/repochanges/main.go:18]: would be nice to have this in luet directly 🙊

**Review by mauromorales — APPROVED** [review]
very cool 😎

**Comment by Itxaka** [comment]
umm doesnt really work properly. Extra versions appear that we cant compare properly and doesnt seem to pick all changes.

Closing for now, if someone wants to take over it, be my guest .

**Pull request closed without merging**